### PR TITLE
feature/add histogram

### PR DESCRIPTION
**Pull request by Superjomn**
- remove the template from Entry so that Entry can set any field type easily.
- add the histogram

### CODE REVIEW

File: server/visualdl/storage_mock.py [hunk 1]
```diff
@@ -35,15 +35,8 @@ def add_image(writer,
...
-    add_scalar("test", "layer/scalar0/max", 1000, 10)
-    add_scalar("valid", "layer/scalar0/max", 1000, 10)
-
-    add_image("train", "layer/image0", 7, 10, 1)
-    add_image("test", "layer/image0", 7, 10, 3)
+def add_histogram(writer, mode, tag, num_buckets):
+    with writer.mode(mode) as writer:
+        histogram = writer.histogram(tag, num_buckets)
+        for i in range(10):
+            histogram.add_record(i, np.random.normal(0.1 + i * 0.01, size=1000))
```
Comment (daming-lu) [inline server/visualdl/storage_mock.py:42]: 这里需不需要 set seed，这样每次产生的mock都是一致的？

Comment (Superjomn) [reply server/visualdl/storage_mock.py:42]: 赞同

**Comment by jacquesqiao** [comment]
please update the code with develop

**Review by jacquesqiao — APPROVED** [review]
LGTM

**Pull request merged**